projects/canopy/src/lib/forms/radio/docs/radio/guide.stories.mdx

```diff
@@ -0,0 +1,105 @@
+
+### Do
+
+![do](docs/forms/radio/do.png)
+
+1. **Do** use when only one item can be selected from a list.
+2. **Do** lay out the options vertically, with one choice per line.
+
+### Don't
+
+![dont](docs/forms/radio/dont.png)
+
+1. **Don't** use when multiple items from a list can be selected, instead use checkboxes.
+2. **Don't** use radio buttons when the preset options is a long list (more than 5) and space is limited, use the select component instead.
+
+## Additional developer details
+
+### LgRadioGroupComponent inputs
+
+| Name     | Description                                                                    | Type        | Default                               | Required |
+|----------|--------------------------------------------------------------------------------|-------------|---------------------------------------|----------|
+| `id`     | HTML ID attribute, auto generated if not provided                              | `string`    | `lg-radio-group-id-${nextUniqueId++}` | No       |
+| `name`   | Set the name value for all inputs in the group, auto-generated if not provided | `string`    | `lg-radio-group-${nextUniqueId++}`    | No       |
+| `value`  | Set the default checked radio button, must match the value of the radio button | `boolean` \ | `string`                              | `null`   | No       |
+| `focus`  | Set the focus on the fieldset                                                  | `boolean`   | `null`                                | No       |
+| `inline` | If true, displays the radio buttons inline rather than stacked                 | `boolean`   | `false`                               | No       |
+
+### LgRadioButtonComponent
+
+#### Inputs
+
+| Name              | Description                                                                                                                   | Type        | Default                             | Required |
+|-------------------|-------------------------------------------------------------------------------------------------------------------------------|-------------|-------------------------------------|----------|
+| `id`              | HTML ID attribute, auto generated if not provided                                                                             | `string`    | `lg-radio-button-${++nextUniqueId}` | No       |
+| `name`            | HTML name attribute                                                                                                           | `string`    | `null`                              | Yes      |
+| `value`           | HTML value attribute                                                                                                          | `string`    | `null`                              | Yes      |
+| `ariaDescribedBy` | HTML ID for the corresponding element that describes the radio, if not provided it will use the hint field where appropriate. | `boolean`   | `null`                              | No       |
```

Comment:
```suggestion
| `ariaDescribedBy` | HTML ID for the corresponding element that describes the radio, if not provided it will use the hint field where appropriate | `boolean`   | `null`                              | No       |
```

Other rows don't use full stops